it/firefox_privacy_notice.md

```diff
@@ -26,7 +26,12 @@ Le ricerche in Firefox possono essere eseguite da diversi punti, come la barra d
 
 * __Dati sulla posizione__: al primo utilizzo di Firefox, l'indirizzo IP dell'utente viene usato per configurare il provider di servizi di ricerca predefinito in base al Paese. [Per saperne di più](https://support.mozilla.org/kb/change-your-default-search-settings-firefox).
 
-* __Query di ricerca__: se viene selezionato un provider che supporta i suggerimenti di ricerca, per impostazione predefinita, Firefox invia le richieste al provider di servizi di ricerca per aiutare l'utente a scoprire frasi comuni usate nelle ricerche da altri e migliorare l'esperienza di ricerca. [Ulteriori informazioni](https://support.mozilla.org/kb/search-suggestions-firefox) che includono come disattivare questa funzione. Se viene abilitato Firefox Suggest, noi e i nostri partner potremmo anche ricevere le query di ricerca dell'utente. [Ulteriori informazioni di seguito](#searches). 
+* __Query di ricerca__: se viene selezionato un provider che supporta i suggerimenti di ricerca, per impostazione predefinita, Firefox invia le richieste al provider di servizi di ricerca per aiutare l'utente a scoprire frasi comuni usate nelle ricerche da altri e migliorare l'esperienza di ricerca. [Ulteriori informazioni](https://support.mozilla.org/kb/search-suggestions-firefox) che includono come disattivare questa funzione. Se viene abilitato I link ai nostri fornitori di servizi di ricerca predefiniti sono:

```

Comment:
This translation is wrong, looks like nobody reviewed the translated text? Please talk with the translator, this is not acceptable for a legal document.

There's a half translation from the previous text (`Se viene abilitato`).

Reply:
Fixed.